packages/turf-line-offset/index.d.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference types="geojson" />
+
+type LineString = GeoJSON.Feature<GeoJSON.LineString>;
+
+/**
+ * http://turfjs.org/docs/#lineOffset
+ */
+declare function lineOffset(line: LineString | GeoJSON.LineString, distance: number, units?: string): LineString;
```

Comment:
Recently I've been adding the `Units` types pulled from `@turf/helpers`.
https://github.com/Turfjs/turf/blob/0cfbda88456a89c7cd988ac17afd6056a42f59a8/packages/turf-rhumb-distance/index.d.ts